packages/zip-it-and-ship-it/src/runtimes/node/bundlers/zisi/resolve.ts

```diff
@@ -31,11 +29,7 @@ const BACKSLASH_REGEXP = /\\/g
 export const resolvePackage = async function (moduleName: string, baseDirs: string[]): Promise<string> {
   try {
     return await resolvePathPreserveSymlinks(`${moduleName}/package.json`, baseDirs)
-  } catch (error) {
-    if (semver.lt(nodeVersion, REQUEST_RESOLVE_MIN_VERSION)) {
-      throw error
-    }
-
+  } catch (_error) {
     try {
       return resolvePathFollowSymlinks(`${moduleName}/package.json`, baseDirs)
     } catch (error_) {
```

Comment:
After your changes, we no longer need to catch the first error, and we can rename the second caught error now that there aren't two to manage:
```suggestion
    } catch {
      ...
      } catch (error) {
        return await resolvePackageFallback(..., error)
```